src/subs.jl

```diff
@@ -102,7 +102,7 @@ Convert a SymEngine numeric value into a number
 
 """
 N(b::Basic) = N(BasicType(b))
-N(b::BasicType{Val{:Integer}}) = eval(parse(toString(b)))  ## HACKY
+N(b::BasicNumber) = eval(parse(toString(b)))  ## HACKY
```

Comment:
The `eval(parse(...))` trick is a way to get integers to parse as a type that depends on the size (so that BigInts) aren't always used. Julia's parser does this, so there may be a more direct way, but I don't know it. However, for floating point numbers this seems to always return a Float64 value. If that is the case, it might be more idiomatic to just call `parse(Float64, string(b))`.  But, in general,  is there a better way to convert a Basic value to a numeric value in SymEngine, then pass that back to Julia.